.github/workflows/c-cpp.yml

```
@@ -6,74 +6,138 @@ on:
-    - name: Upgrade Clang
-      run: |
-        sudo apt-get install clang clang-21 lld lld-21 llvm-21-linker-tools || true
-
-    - name: Add dl.winehq.org
-      run: |
-        sudo dpkg --add-architecture i386
+    - name: Set build config
+      id: cfg
+      env:
+        suffix: ${{ matrix.os == 'windows' && '.exe' || '' }}
+      run: |
```

actionlint (1.7.4)

workflow command "set-output" was deprecated. use `echo "{name}={value}" >> $GITHUB_OUTPUT` instead: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions

(deprecated-commands)